roles/add_day2_node/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
         var: discovered_host
         verbosity: 1
 
-    - name: "Wait for 20 min until {{ inventory_hostname }} to be ready"
+    - name: "Wait for 20 min until {{ inventory_hostname }} waiting for requirements checks to complete"
```

Comment:
Not a blocker but what do you think of this instead.
`Waiting for {{ inventory_hostname }} to complete requirements checks. This can take up to 20 minutes.`

Reply:
We seem to use "Wait up to X min for" in most places so I'll change them to match that format

Reply:
ack +1